docs/UserGuide.md

```diff
@@ -599,4 +604,23 @@ This is what to expect when you type in the `help` command or press `F1`:
    <strong><i>{End of Appendix A, please return to the <a href="#table-of-contents">Table of Contents</a> } </i
    ></strong> 
 </p>
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Appendix B: International Standards Followed 
+
+### Country Codes
+
+**_TBM_** follows 2-letter country codes that follow the ISO3166 specification.
+[Here's the Official Search tool](https://www.iso.org/obp/ui/#search) for country codes.
+
+### Phone Numbers
+
+**_TBM_**'s format for phone numbers is **_based_** on the [E.164 standard](https://en.wikipedia.org/wiki/E.164) as recommended by the _ITU Telecommunication Standardization Sector (ITU-T)_.
+**_However_**, **_TBM_** doesn't allow for spaces in between phone number fields. For example, `+65 8186 7742` may follow this international, but **_TBM_ requires you to remove spacing within the phone number field** so the correct way to input it would be `+65 81867742`. 
```

Comment:
grammatical error: "may follow this international"

Reply:
woops missed out a "standard" there